frontend/gist-frontend/src/api/api-handler.ts

```diff
@@ -3,27 +3,35 @@ import { local_url } from "./urls";
 export async function APIHandler(
   extension: string,
   method: string,
-  body?: object
+  body?: object,
+  info?: string
 ) {
   let response = "";
-  if (method === "GET") {
-    await fetch(`${local_url}/${extension}`)
+  let data = "";
+  if (!!body) {
+    data = JSON.stringify(body);
+  }
```

Comment:
I'm seeing a lot of this !! stuff. What does it mean? I've looked up a definition but didn't really make much sense for some reason. How'd you come about this syntax?

Reply:
it means `if(body !== undefined)